analytics/js_tracking_api.rst

```diff
@@ -0,0 +1,924 @@
+    }]);
+
+.. function:: getCustomVariable(index, scope)
+
+    :param number index: **Required** Number from 1 to 5 where variable is stored
+
+    :param string scope: **Required** Scope of the variable, "visit" or "page"
+
+Example of usage::
+
+    _paq.push([ function() {
+        var customVariable = this.getCustomVariable(1, "visit" );
+        console.log(customVariable);
+    }]);
+
+Custom Dimensions
+^^^^^^^^^^^^^^^^^
+
+Tracking Custom Dimension
+`````````````````````````
+If you want to set custom dimension to use it in tracking functions use ``setCustomDimension`` function::
+
+    _paq.push(["setCustomDimension", customDimensionID, customDimensionValue]);
+
+.. data:: customDimensionID
+
+    **Required** Id of dimension
+
+.. data:: customDimensionValue
+
+    **Required** Value of Custom Dimension
+
+.. warning::
+
+    When you set Custom Dimension that value will be used in all tracking requests within page load.
+
+Example of usage::
+
+    _paq.push(["setCustomDimension", 3, "loginStatus"]);
+
+
+Retrieving Custom Dimension
+```````````````````````````
+.. todo::
+It would be nice to have some examples of returned data.
+
+You can access custom dimension by providing function that will use ``getCustomDimension`` function::
+
+    _paq.push([ function() {
+        var customDimension = this.getCustomDimension(index);
+    }]);
+
+.. function:: getCustomDimension(index, scope)
+
+    :param number index: **Required** Index of custom dimension
+
+Example of usage::
+
+    _paq.push([ function() {
+        var customDimension = this.getCustomDimension(1);
+        console.log(customDimension);
+    }]);
+
+Content Tracking
+^^^^^^^^^^^^^^^^
+
+Content Tracking is an useful feature when you want to measure performance of your website or app. For example you can check
+how many times your file was downloaded, or banner was seen, how many users triggered modal screen with some promotion and then
+entered products etc.
+
+To track content, it has to have ``data-track-content`` css class attached to it.
+
+Tracking all content impressions within a page
+``````````````````````````````````````````````
+To track all content impression you can use ``trackAllContentImpressions`` function. If this function will be invoked
+multiple times it won't send duplicated data unless ``trackPageView`` was used between invocations::
+
+    _paq.push(["trackAllContentImpressions"]);
+
+Tracking all visible content impressions
+````````````````````````````````````````
+To track all visible content impressions you can use ``trackVisibleContentImpressions`` function.
+
+Code::
+
+    _paq.push(["trackVisibleContentImpressions", checkOnScroll, timeIntervalInMs]);
+
+.. data:: checkOnScroll
+
+    **Required** If set to true it will invoke this function to track new visible content impressions on scroll event.
+    (It won't detect content blocks placed in a scrollable element)
+
+.. data:: timeIntervalInMs
```

Comment:
I don't like including time unit in the name. It should be explained in description. Name should just convey intention. Maybe `watchInterval` or `interval`?